editor/filesystem_dock.cpp

```diff
@@ -1377,15 +1377,15 @@ void FileSystemDock::_get_all_items_in_dir(EditorFileSystemDirectory *p_efsd, Ve
 	}
 }
 
-void FileSystemDock::_find_file_owners(EditorFileSystemDirectory *p_efsd, const Vector<String> &p_renames, Vector<String> &r_file_owners) const {
+void FileSystemDock::_find_file_owners(EditorFileSystemDirectory *p_efsd, const Vector<String> &p_renames, HashSet<String> &r_file_owners) const {
```

Comment:
```suggestion
void FileSystemDock::_find_file_owners(EditorFileSystemDirectory *p_efsd, const HashSet<String> &p_renames, HashSet<String> &r_file_owners) const {
```
This one is being checked for content, you changed it to `Vector` before which slows things down considerably here

Reply:
I was wrong about what I had changed before it seems

Reply:
This method was renamed in a PR after yours, it used to be `_find_remaps`